### ISSUE TITLE
R TabExercise cannot be run without installing dplyr in requirements.sh

### ISSUE THREAD

**Issue by machow** [datacamp/authoring#76]
Feel free to close, if not an issue :). A reader who copies the R TabExercise example into Teach will see the following.

![image](https://user-images.githubusercontent.com/2574498/35997237-e7e37442-0ce6-11e8-8ac0-37bc18295c05.png)

They would need to install dplyr in their `requirements.sh` to get the exercise to run. It might be helpful to either 

1. modify the example so that it can run with the base R packages, or
2. change the base package to include dplyr

option (2) is probably part of a bigger discussion with @rv2e (maybe as a content-tech-request?)

**Comment by ramnathv** [comment]
@machow Good point. To make sure I understand, you are suggesting that the example provided only use packages that ship with our base Docker image so that they can be copy-pasted without issues?

**Comment by machow** [comment]
@ramnathv that's right. I suspect that if a user pastes in the example, and it doesn't run, they will get confused / frustrated.

The authoring docs are super convenient, it's been really helpful being able to navigate here as a point of entry for examples!
